### ISSUE TITLE
Checkout, Shipping address modal. Replace "Cancel" with "Close" on empty address book.

### ISSUE THREAD

**Issue by alexanderfc** [FoxComm/highlander#1514]
If customer's address book is empty, "Cancel" should be replaced with "Close" so there wouldn't be that mediate "Address book" modal. I think it's just not needed if address book is empty. UX logic here is:
1. Click "Choose" at "Shipping address" block if you want to add new address -- that will bring up a "new shipping address" form in modal.
2. Click "Close" (where currently "Cancel" is for all cases) to close form if for some reason you don't want to add a new address right now.

In short - this saves customer 1 pointless click (spend on closing empty address book). Pretty cool, huh? :)
